Speekins/stretch-project
# Testing/cypress add fixtures

#### What's this PR do?
- Add files for cypress including fixture files
#### Where should the reviewer start?
-Look at the cypress files, right now I don't have a fixture file to check for an edit to the verified status. We will have to think about how we want the verified status to function and then how to test it
#### How should this be manually tested?
- make sure to run `npm i` then `npm run cypress` the only test currently written is a default test
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
